README.md

```diff
@@ -207,74 +207,78 @@ git clone https://github.com/ggerganov/llama.cpp
 cd llama.cpp
 ```
 
-### Build
-
-In order to build llama.cpp you have three different options.
-
-- Using `make`:
-  - On Linux or MacOS:
-
-      ```bash
-      make
-      ```
-
-  - On Windows:
-
-    1. Download the latest fortran version of [w64devkit](https://github.com/skeeto/w64devkit/releases).
-    2. Extract `w64devkit` on your pc.
-    3. Run `w64devkit.exe`.
-    4. Use the `cd` command to reach the `llama.cpp` folder.
-    5. From here you can run:
+### Build *(simple, without BLAS accelearion)*
```

Comment:
```suggestion
### Build *(simple, without BLAS acceleration)*
```

More generally I don't think "BLAS acceleration" is a good term (although it's already used in the README on master). For GPUs it makes sense because they employ hardware acceleration but with a BLAS library the difference is just from algorithms. I would just call it something like "native CPU only"